src/createServer.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-shadow */
```

Comment:
Using `eslint-disable` is a bag practice. Try to avoid it here and in test tasks. Find the way to solve it without `eslint-disable`.